src/engine/create-changeset/tasks/create-fetch-changed-tasks.ts

```diff
@@ -47,7 +47,10 @@ async function getEntityPatches({
     const targetEntry = targetEntries.find((entry) => entry.sys.id === entryId)
 
     if (sourceEntry && targetEntry) {
-      const patch = createPatch({ targetEntry, sourceEntry })
+      const cleanedSourceEntry = sourceEntry.metadata ? { ...sourceEntry, metadata: undefined } : sourceEntry
```

Comment:
This for some reasons throws on my side a ts error (but that could be my setup 🤔):

```
 create-fetch-changed-tasks.ts    50  46 error    2339   Property 'metadata' does not exist on type 'Entry<any, ChainModifiers, string> | ContentType'.
   Property 'metadata' does not exist on type 'ContentType'. (lsp)
```

Reply:
Yes, that error had been there before, it's not related to my change. It's basically because of the generic (which can be either Entry or ContentType)